src/components/List/List.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2019, SumUp Ltd.
```

Comment:
Can we update the copyright to 2020?

Reply:
Since the initial code was written in 2019, it should remain as 2019.